.pre-commit-config.yaml

```diff
@@ -111,3 +111,14 @@ repos:
     - commit
     - push
     - manual
+- repo: local
+  hooks:
+  - id: line-length
+    name: line-length
+    description: Check for lines longer 100 and brakes them before 80.
+    entry: ./scripts/line-length.py
```

Comment:
According to PEP8 Python modules, can use an underscore to improve readability, not a dash.